interfaces.go

```diff
@@ -101,15 +102,15 @@ func (r Request) FromHTTPRequest(request *http.Request) Request {
 	r.QueryString = request.URL.RawQuery
 
 	// Body
-	if request.GetBody != nil {
-		if bodyCopy, err := request.GetBody(); err == nil && bodyCopy != nil {
-			body, err := ioutil.ReadAll(bodyCopy)
-			if err == nil {
-				r.Data = string(body)
-			}
+	if request.Body != nil {
+		bodyBytes, err := ioutil.ReadAll(request.Body)
+		_ = request.Body.Close()
+		if err == nil {
```

Comment:
If `err != nil`, it might still be possible that some data was read.

> [ReadAll](https://golang.org/pkg/io/ioutil/#ReadAll) reads from r until an error or EOF and returns the data it read.

Wouldn't it be more beneficial to always create the buffer with the data that was read, and only log the error?

E.g. the client could abruptly close the connection while the server is reading the body.

Reply:
Tracking improvement in #117.

Reply:
This introduces a new bug. If `ioutil.ReadAll` returns an error the body is left in a closed state, so reading the body later results in a `http.ErrBodyReadAfterClose`, so instead of the original error getting sent to Sentry you end up just getting `http: invalid Read on closed Body` with no way of knowing what the original error was.

Reply:
@faulkner solid. I'll take this into account when I work on #117, which should be some time next week.
Contributions welcome as well, just send us a patch if you have one. Thanks!